translation/en/RedirectedResources/assets/abdata/adv/scenario/c06/00/80/translation.txt

```diff
@@ -5,4 +5,4 @@
 「たしかに精力がつきそうなものばかりですね」=“Certainly, it is made up of only things that do empower one's libido.”
 「いや、単純にメニューの中で濃い味付けなのと、量が多いからなんだけど……」=“No, I mean, for me it's just that it's the most spiced dish out of all of them here, the serving's also huge, that's all...”
 「そ、そうでしたか。それは、その、失礼いたしました」=“I-I see. I, uhm, I do apologize.”
-（「殿方＝エロい」という先入観はどうにかしないとなぁ。いや、間違ってはないんだけどね）=(I really gotta do something about her just assuming that men)
+（「殿方＝エロい」という先入観はどうにかしないとなぁ。いや、間違ってはないんだけどね）=(I really gotta do something about her just assuming that men = lewd. No, well, not like she's wrong about it though.)
```

Comment:
Does the second `=` need escaping?  Even if it doesn't need it, does it work if it is to keep it clear?

Reply:
I'm not sure how it works in current AT version, it's been changed a couple of times. I'd still avoid having more than one = in the line to avoid issues with scripts.
```suggestion
（「殿方＝エロい」という先入観はどうにかしないとなぁ。いや、間違ってはないんだけどね）=(I really gotta do something about her just assuming that men equal lewd. No, well, not like she's wrong about it though.)
```

Reply:
Changing `=` to `equal` here is fine, go with that.